src/server/lock_manager/waiter_manager.rs

```diff
@@ -201,6 +206,7 @@ impl Waiter {
             delay: Delay::new(deadline),
             diag_ctx,
             _lifetime_timer: WAITER_LIFETIME_HISTOGRAM.start_coarse_timer(),
+            wait_timer,
```

Comment:
I think `wait_timer` duplicates with `_life_timer`. They may be combined into one.

It seems the place you create `wait_timer` records the time more accurately.

Reply:
> I think wait_timer duplicates with _life_timer. They may be combined into one.

Good idea~I have combined them together. 

> It seems the place you create wait_timer records the time more accurately.

I'm conservative in metric, not wanting to ignore the Runtime overhead.